src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,27 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  let bulls = 0;
+  let cows = 0;
+
+  const userInputStr = String(userInput);
+  const numberToGuessStr = String(numberToGuess);
+
+  for (let i = 0; i < userInputStr.length; i++) {
+    const guessNumber = userInputStr[i];
+    const rightNumber = numberToGuessStr[i];
+
+    if (guessNumber === rightNumber) {
+      bulls += 1;
+    } else if (
+      numberToGuessStr.includes(guessNumber) &&
+      guessNumber !== rightNumber
```

Comment:
The current logic for counting cows does not account for the possibility of counting the same digit multiple times if it appears more than once in the `numberToGuessStr`. Consider using a mechanism to track which digits have already been counted as cows to avoid overcounting.